lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -56,66 +58,39 @@ def self.get_release_version(product_name:)
           return { VERSION_NAME => name, VERSION_CODE => code }
         end
 
-        def self.properties_file_exists
-          properties_file_path = File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', 'version.properties')
-
-          return File.exist?(properties_file_path)
+        def self.version_properties_file
+          File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', 'version.properties')
         end
 
         # Extract the version name and code from the `version.properties` file in the project root
         #
-        # @param [String] product_name The name of the app to extract the version from e.g. wordpress, simplenote
```

Comment:
This is technically a breaking change – should we still include this param but mark it deprecated until it's removed like the ones above?

Reply:
It's a private method only called from the other, public ones in this helper file, hence the removal here that doesn't affect the public API unlike the others